src/test/java/com/github/cameltooling/lsp/internal/codeactions/UnknownPropertyQuickfixTest.java

```diff
@@ -31,6 +31,7 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import org.awaitility.core.ConditionFactory;
```

Comment:
I don't see any dependency upgrades. Was that library already included?

Reply:
yes
https://github.com/camel-tooling/camel-language-server/blob/81368e772e67102240f8372868b5b6b1765481f7/pom.xml#L264-L269